src/components/Map/OrganizationMap.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Map from './Map';
+
+const googleMapKey = 'AIzaSyAwKdrqS2GfCt9b2K1wAopDc9Ga0N1BVUM';
+const googleMapURL = `https://maps.googleapis.com/maps/api/js?key=${googleMapKey}&v=3.exp&libraries=geometry,drawing,places`;
+
+const defaultZoom = 12;
+const defaultCenter = { lat: 42.3731, lng: -71.0162 };
+
+class OrganizationMap extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      center: this.props.center ? this.props.center : defaultCenter,
+      zoom: defaultZoom,
+    }
+  }
+
+
+  markerHover = (key, event) => {
+    event.map.getCanvas().style.cursor = 'pointer';
+    this.setState({
+      hoveredItem: key
+    });
+  }
+
+  markerEndHover = (key, event) => {
+    event.map.getCanvas().style.cursor = '';
+    this.setState({
+      hoveredItem: ''
+    });
+  }
+
+  onOrganizationClick = (key, event) => {
+    const organizationZoom = 11;
+    const org = this.props.organizations.find(o => o.id === key);
+
+    this.setState({
+      center: [org.position.longitude, org.position.latitude],
+      zoom: [organizationZoom]
+    });
+  }
+
+  setOpenMarker = index => {
+
+    Object.entries(this.props.locationAddressHashTable).forEach(([index2, orgRef])  => {
+
```

Comment:
I can't quite tell what this is going to be looping over. Perhaps turn the `Object.entries` call into a variable first to indicate  that

Reply:
Deprecated code at the moment. The setOpenMarker was to be a function that would zoom in to a marker that was clicked from the ResultList. I'm not sure how it was implemented as I have refactored the locatioinAddressHashTable. I'm planning to create another issue to rectify this broken functionality. I thought I leave this here for future reference but this function doesn't go anywhere right now.